### ISSUE TITLE
U cant talk in server console.

### ISSUE THREAD

**Issue by RebornTrack970** [jakubDoka/The_worst_mindustry_plugin_2#6]
This plugin makes the server unable to talk in the server console.

**Comment by jakubDoka** [comment]
Does it relate to "say" command not working?

**Comment by RebornTrack970** [comment]
Yes, we can talk however, it doesnt show on the screen with blue and it annoys a bit can u fix it also, server crashed 5 times so i deleted the plugin

**Comment by RebornTrack970** [comment]
[05-22-2020 | 18:22:33] [I] something horrible happen
[05-22-2020 | 18:22:33] [I] something horrible happen
[05-22-2020 | 18:22:33] [I] something horrible happen
[05-22-2020 | 18:22:33] [I] something horrible happen

**Comment by RebornTrack970** [comment]
And crash...
